provisioner/roles/manage_ec2_instances/defaults/main.yml

```diff
@@ -70,3 +69,19 @@ ec2_info:
     architecture: x86_64
     filter: 'F5 Networks Prelicensed Hourly BIGIP-13.1*Good 25MBPS*'
     username: admin
+  splunk_enterprise:
+    owners: 309956199498
+    size: t2.micro
+    os_type: linux
+    disk_space: 10
+    architecture: x86_64
+    filter: 'RHEL-7.6_HVM-20190515-x86_64-0-Access2-GP2'
+    username: ec2-user
+  checkpoint_firewall:
```

Comment:
Before the firewall we first need the cloud security management server. The following details are for the management server, I can work on the firewalls as soon as we have this PR merged.